README.md

```diff
@@ -1,2 +1,197 @@
 # planship-nuxt
-Planship Nuxt module
+
+Welcome to `@planship/nuxt`, a [Nuxt module](https://nuxt.com/modules) that makes [Planship](https://planship.io) intergration with [Nuxt](https://nuxt.com) apps a breeze. This module is based on the [`@planship/vue`](https://github.com/planship/planship-vue) plugin and [`@planship/fetch`](https://github.com/planship/planship-js/tree/master/packages/fetch) library.
+
+A complete, working example of a Nuxt app that used the `@planship/nuxt` module can be found at https://github.com/planship/planship-nuxt-demo
+
+# Getting started
+
+Install `@planship/nuxt` with npm, yarn or pnpm:
+
+```sh
+npm install @planship/nuxt
+# or
+yarn add @planship/nuxt
+# or
+pnpm add @planship/nuxt
+```
+
+Then, add `@planship/nuxt` to the `modules` section of your `nuxt.config.ts`:
+
+```ts
+export default defineNuxtConfig({
+  modules: ['@planship/nuxt'],
+})
+```
+
+Finally, configure your Planship product slug and authentication credentials inside `defineNuxtConfig` or via the environmental variables.
+
+```ts
```

Comment:
Consider adding an "or" followed by an example .env. Might consider separating the two into a couple sub-sections if it's too confusing.

Reply:
I already added a more complete `defineNuxtConfig` snippet.